programs/drift/src/math/repeg.rs

```diff
@@ -252,17 +252,25 @@ pub fn calculate_per_peg_cost(
     terminal_quote_asset_reserve: u128,
 ) -> DriftResult<i128> {
     // returns a signed per_peg_cost relative to delta peg
+    // signed means that "cost" to amm is influenced whether delta_peg is the same sign
+
     let per_peg_cost = if quote_asset_reserve != terminal_quote_asset_reserve {
         quote_asset_reserve
             .cast::<i128>()?
             .safe_sub(terminal_quote_asset_reserve.cast::<i128>()?)?
-            .safe_div(AMM_RESERVE_PRECISION_I128 / PEG_PRECISION_I128)?
-            .safe_add(1)?
+            .safe_div_ceil(AMM_RESERVE_PRECISION_I128 / PEG_PRECISION_I128)?
```

Comment:
@crispheaney this will make negative number "larger negative" than otherwise?

Reply:
it rounds to positive infinity